recipes/code-aster/meta.yaml

```
@@ -0,0 +1,111 @@
+{% set name = "code-aster" %}
+{% set version = "14.6" %}
+{% set minor_version = "11" %}
+{% set sha256 = "107ed98d10bc05866bdb3053f088fb0f51276ea14f7b615ec0d410a4b147ef12" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  - fn: {{ name }}_{{ version }}.{{ minor_version }}.tar.gz
+    url: https://gitlab.com/codeaster/src/-/archive/{{ version }}.{{ minor_version }}/src-{{ version }}.{{ minor_version }}.tar.gz
```

```suggestion
{% set name = "code-aster" %}
{% set version = "14.6.11" %}
{% set sha256 = "107ed98d10bc05866bdb3053f088fb0f51276ea14f7b615ec0d410a4b147ef12" %}

package:
  name: {{ name|lower }}
  version: {{ version }}

source:
  - fn: {{ name }}_{{ version }}.tar.gz
    url: https://gitlab.com/codeaster/src/-/archive/{{ version }}/src-{{ version }}.tar.gz
```
Why would you split the version into two variables? The package version should be the full version.